packages/@okta/vuepress-site/guides/sign-into-spa/configure-the-sdk/index.md

```diff
@@ -14,9 +14,14 @@ const config = {
   clientId: '{clientId}',
   issuer: 'https://{yourOktaDomain}.com/oauth2/default',
   redirectUri: 'http://localhost:8080/implicit/callback',
+  scope: 'openid profile email',
 };
 ```
 
+> Note: You can also get the issuer config using this link `https://{yourOktaDomain}.com/oauth2/default/.well-known/openid-configuration`
```

Comment:
I don't think this is valuable. If a reader can construct this URL, they already can construct `https://{yourOktaDomain}.com/oauth2/default`.
Our site will automatically replace `{yourOktaDomain}` for them too, if they are signed in.